### PR TITLE
CO-52:Frontend dungeon event list

### PR DESCRIPTION
**Pull request by Galicz555**

### CODE REVIEW

File: src/app/components/hero-action/hero-action.component.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-hero-action',
+  templateUrl: './hero-action.component.html',
+  styleUrls: ['./hero-action.component.scss']
+})
+export class HeroActionComponent implements OnInit {
+  private _events: string[] = ['Love and die', 'Fuck and drink', 'Live and kill while sacrificing everything'];
```
Comment (juhaszPeterDev) [inline src/app/components/hero-action/hero-action.component.ts]: keep this variable empty. if there is no battle event, this should be shown, so keep this array empty. :)

File: src/app/components/register/register.component.spec.ts [hunk 2]
```diff
@@ -19,6 +19,8 @@ import { NavBarComponent } from '../nav-bar/nav-bar.component';
 import { SingleHeroPageComponent } from './../single-hero-page/single-hero-page.component';
```
Comment (juhaszPeterDev) [inline src/app/components/register/register.component.spec.ts]: you could just neuter the .spec file as in other places. Pls follow the company policy.

File: src/app/components/hero-action/hero-action.component.ts (hunk 1 above)
Comment (Galicz555) [reply src/app/components/hero-action/hero-action.component.ts]: solved

File: src/app/components/register/register.component.spec.ts (hunk 2 above)
Comment (Galicz555) [reply src/app/components/register/register.component.spec.ts]: solved

**Pull request merged**